nvflare/private/fed/server/fed_server.py

```diff
@@ -808,6 +808,7 @@ def stop_training(self):
 
     def fl_shutdown(self):
         self.engine.stop_all_jobs()
+        self.engine.stop_job_runner()
```

Comment:
self.engine.stop_all_jobs() will shutdown all the running jobs. This should not be called when the server turning to Cold, otherwise the running won't be able to switch to new SP and keep running. Instead, we need to work on a new function to stop all the running job processes, but however, not to impact the snapshot storage and job status storage.